src/ixdat/readers/zilien.py

```diff
@@ -268,6 +275,49 @@ def _read_metadata(file_handle):
 
         return metadata, series_headers, column_headers
 
+    def _get_technique(self):
+        """Get technique according to parsed series headers and column headers.
+
+        This method covers the following cases:
+          - "pot" and "EC-lab" is in the metadata and in the headers. There is "EC-lab" data.
+          - "pot" and "EC-lab" is in the metadata and in the headers. There is no "EC-lab" data. (bug)
+          - "pot" is not in the metadata, but it is in the headers and the data part is filled with NaN. (bug)
+          - "pot" is neither in the metadata, nor in the headers.
+
+        """
+        pot_in_metadata = "pot_pot_count" in self._metadata
+        pot_in_series_headers = "pot" in self._series_headers
+
+        result = ""
+        # there was EC measurement
+        if pot_in_metadata and pot_in_series_headers:
+            result = "EC-MS"
+
+            # BUG CASE: EC-lab series header gets included, but .mpt data don't
+            if (
+                len(self._column_headers) < len(self._series_headers)
+                and self._series_headers[-1] == "EC-lab"
+            ):
+                result = "MS"
+                sys.stderr.write(
```

Comment:
Dumb question:
Where does this text go? Does it get included in an error message somewhere?

Reply:
It is printed in a terminal through STDERR (standard error) pipe where errors are reported. STDOUT (standard output) pipe is used for normal text and messager.
I considered using `userwarnings`, but I don't like how it adds the source code line.